front/src/gc-components/GCCheckbox.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from "react";
+import { Checkbox, CheckboxProps, MantineSize } from "@mantine/core";
+import { MantineNumberSize } from "@mantine/styles/lib/theme/types/MantineSize";
+
+interface GCCheckboxProps extends CheckboxProps{
+    children: ReactNode;
+    ml?: number;
+    size?: MantineSize,
+    className?: string;
+    mt?: MantineNumberSize;
+}
+
+export const GCCheckbox = ({ children, ml, className, size, checked, mt, ...props }: GCCheckboxProps) => {
+    return (
+        <Checkbox ml={ml} className={className} mt={mt} {...props}>
+            {children}
```

Comment:
which child will checkbox have? is it needed here?

Reply:
like text/list?

Reply:
Ok, np